src/server.js

```diff
@@ -1 +1,129 @@
-// Write your code here
\ No newline at end of file
+const express = require('express');
+const fs = require('fs');
+let moviesList = require('../data.json');
+const uuid = require('uuid/v4'); 
+const bodyParser = require('body-parser'); 
+
+const app = express();
+const port = 4000;
+
+getTitle = (obj) => obj.map(item => item.title);
+
+sortByWord = (obj) => obj.sort((item1,item2) => item1.localeCompare(item2));
```

Comment:
it is array, not an obj
will be better to name it textSort etc.
(item1,item2) - should be space after comma